docs/cpp/faq-cpp.md

```diff
@@ -139,6 +133,6 @@ If your debugger is showing a grey stack trace, won't stop at a breakpoint, or t
 
 ## What do I do if I suspect a C/C++ extension problem
 
-If you are experiencing a problem with the extension that we can't diagnose based on information in your issue report, we might ask you to enable logging and send us your logs. See [C/C++ extension logging](/docs/cpp/enable-logging-cpp.md) for how to get C/C++ extension logs.
+If you have any other questions, please start a discussion at [GitHub discussions](https://github.com/microsoft/vscode-cpptools/discussions), or if you believe your issue may require a fix, file an issue at [GitHub issues](https://github.com/microsoft/vscode-cpptools/issues).
```

Comment:
it would be better to rephrase
"or if you believe your issue may require a fix", file an issue at" -> "or for any issues, file an issue at"

Reply:
I don't understand. "for any issues" seems too vague since their issue could just be a question such as how to get their IntelliSense working. The current wording differentiates between issues that are questions that are not expected to require a fix (best handled in "discussinos") and issues that require a fix (best handled in "issues").

Reply:
Did you have another alternative wording?

Reply:
The format I wrote the fix in might be confusing - I was looking to make the phrasing more clear because to a user, any issue they find they will believe will require a fix, so using the word "believe" is redundant. I agree that we should mention the discussions first if they are confused on how to use the product, and issues second. Here is the full sentence of what I meant:
"If you have any other questions, please start a discussion at [GitHub discussions](https://github.com/microsoft/vscode-cpptools/discussions), or if you find an issue, file an issue at [GitHub issues](https://github.com/microsoft/vscode-cpptools/issues)."

Reply:
Ok, that sounds good to me.